README.md

```diff
@@ -1 +1,156 @@
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Awesome books <a name="about-project"></a>
+
+In this project, I built a simple HTML list of To Do tasks. The list was styled according to the specifications provided. This simple web page is built using webpack and served by a webpack dev server.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-html">HTML</a></li>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-css">CSS</a></li>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-javascript">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Follow a correct Gitflow**
+- **Comply with linters**
+- **DOM events with Javascript**
+- **Javascript events**
+- **Use objects to store and access data**
+- **Use localStorage**
+- **Use webpack to bundle JavaScript**
+- **Use ES6 modules to write modular JavaScript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]( https://eamador1.github.io/to-do-list/)
+  https://eamador1.github.io/to-do-list/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+✅ Github account <br>
+✅ Visual Studio Code installed <br>
+✅ Node.js installed <br>
+✅ Git Bash installed (optional)   
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd [my-folder]
+  git clone git@github.com:eamador1/awesome-library.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+**Erwin Amador**
+- GitHub: [@ErwinAmador](https://github.com/eamador1)
+- Twitter: [@ErwinAmador](https://twitter.com/hokieamador)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Use libraries and frameworks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+## 👤 Collaboration:
+
+Special thaks to Damian Div Bustros who was my coding partner for Awesome Books Project:
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
![image](https://github.com/eamador1/to-do-list/assets/93039359/d857cd35-f203-486c-a3ff-0faeafa8c1c7)


**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you move the highlighted information in the **Contributing** section cuz I think you mistakenly merged both sections 😄 Kindly put the highlighted lines in the **Contributing** section to make your readme descriptive and professional 👍